package.json

```diff
@@ -17,7 +17,8 @@
     "test:routes": "NODE_ENV=mocha LOG_LEVEL=OFF mocha --compilers ts:ts-node/register -r tsconfig-paths/register $(find src/test \\( -name '*.ts' -or -name '*.js' \\) -path '*/routes/*') --reporter mochawesome --reporter-options reportFilename=routes,inlineAssets=true,reportTitle=legal-rep-frontend-routes",
     "test:a11y": "mocha --compilers ts:ts-node/register -r tsconfig-paths/register src/test/a11y --timeout 60000 --reporter mochawesome --reporter-options reportFilename=a11y,inlineAssets=true,reportTitle=legal-rep-frontend-a11y",
     "test:nsp": "nsp check",
-    "test:coverage": "cross-env NODE_PATH=. nyc istanbul cover  ./node_modules/mocha/bin/_mocha -- --compilers ts:ts-node/register -r tsconfig-paths/register $(find src/test \\( -name '*.ts' -or -name '*.js' \\) ! -path '*a11y*' ! -path '*/routes/*')",
+    "test:coverage": "cross-env NODE_PATH=. nyc istanbul cover  ./node_modules/mocha/bin/_mocha -- --compilers ts:ts-node/register -r tsconfig-paths/register $(find src/test \\( -name '*.ts' -or -name '*.js' \\) ! -path '*a11y*')",
```

Comment:
```
cross-env NODE_PATH=. nyc istanbul cover  mocha ...
```

Reply:
what's the error?

Reply:
Something like this: so i have used full path for running from windows machine for mocha and when i finished my local test updated back to as shown above

C:\cmc\cmc-legal-rep-frontend\node_modules\.bin\mocha.CMD:1
(function (exports, require, module, __filename, __dirname) { @IF EXIST "%~dp0\node.exe" (
                                                              ^

SyntaxError: Invalid or unexpected token
    at createScript (vm.js:74:10)
    at runInThisContext (vm.js:116:10)
    at Object.vm.runInThisContext (C:\CMC\cmc-legal-rep-frontend\node_modules\nyc\node_modules\istanbul-lib-hook\lib\hook.js:141:16)
    at Module._compile (module.js:537:28)
    at Module.replacementCompile (C:\CMC\cmc-legal-rep-frontend\node_modules\nyc\node_modules\append-transform\index.js:58:13)
    at module.exports (C:\CMC\cmc-legal-rep-frontend\node_modules\nyc\node_modules\default-require-extensions\js.js:8:9)
    at Object..js (C:\CMC\cmc-legal-rep-frontend\node_modules\nyc\node_modules\append-transform\index.js:62:4)
    at Module._extensions.(anonymous function) (C:\CMC\cmc-legal-rep-frontend\node_modules\istanbul\lib\hook.js:109:37)
    at Object.<anonymous> (C:\CMC\cmc-legal-rep-frontend\node_modules\nyc\node_modules\append-transform\index.js:62:4)
    at Module.load (module.js:507:32)

Reply:
This is the istanbul issue:
https://github.com/gotwarlost/istanbul/issues/13#issuecomment-18175599